### ISSUE TITLE
ArgoCD URL not provided in the practice in advance

### ISSUE THREAD

**Issue by Dim21-star** [mabusaa/argocd-course-issues-tracker#14]
When trying to implement tutorial practice "Diffing Customization, Ignore Differences Demo" there is no link of ArgoCD UI in order to provide my private git repo URL and then to proceed to apply the manifest file. Due to that when I apply the manifest the validation is failing since it cannot read my git repo from the Repositories tab in "Settings"

Could this be corrected in order the link to be provided in advance?

Regards,
Dim

**Comment by mabusaa** [comment]
Hi ,

I understand , when I designed this practice I suppose it will be a public repo, but what you can do is to add the private repo declaratively or CLI (good opportunity to practice ) or I can update this and include the UI link .

Or you can open the UI always using the ports page.  enter the port as 32073 and click access

![image](https://user-images.githubusercontent.com/13915373/222973676-c6870fcd-f229-4fe3-a2bf-1a85922a8039.png)

**Comment by Dim21-star** [comment]
Thank you mabusaa for your prompt reply!

I managed to do it declaratively, by using ArgoCD CLI and from UI.

I had to install arcocd CLI as describe before in your section but here is the list of commands that I used in the terminal:

1. curl -sSL -o /usr/local/bin/argocd https://github.com/argoproj/argo-cd/releases/download/v2.4.8/argocd-linux-amd64
2. chmod +x /usr/local/bin/argocd
3. argocd login 172.30.1.2:32073 --grpc-web --plaintext
4. kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d; echo
5. argocd repo add "my-private-git-repo" --username "username" --password "password"


Best,
Dim21-star
